src/SFML/Window/Unix/InputImpl.cpp

```diff
@@ -194,24 +194,21 @@ bool InputImpl::isMouseButtonPressed(Mouse::Button button)
 
     // Get pointer mask
     xcb_query_pointer_reply_t* pointer = xcb_query_pointer_reply(connection, xcb_query_pointer(connection, XDefaultRootWindow(display)), NULL);
+    uint16_t mask = pointer->mask;
```

Comment:
Look at the original issue, it is all explained. The "break" statements were missing, and instead of just adding them I suggested a better rewrite.